templates/guides/getting-started.md.tmpl

```diff
@@ -80,4 +81,31 @@ output "url" {
 output "token" {
   value = qdrant-cloud_accounts_auth_key.example-key.token
 }
-```
\ No newline at end of file
+```
+
+
+### Available cloud providers and regions
+```yaml
+aws:
+- us-east-1
+- us-west-1
+- eu-central_lega-y
```

Comment:
is this correct, should this one be deleted here?

Reply:
No, it is a mistake. 

Thank you for noticing.